types/node/index.d.ts

```diff
@@ -5110,6 +5110,7 @@ declare module "stream" {
             writable: boolean;
             constructor(opts?: WritableOptions);
             _write(chunk: any, encoding: string, callback: Function): void;
+            _writev(chunks: Array<{chunk: any, encoding: string}>, callback: Function): void;
```

Comment:
You could consider to use `(err?: Error) => void` (also for `write` and in `Duplex`).

Reply:
Sounds good. Done.